test/spec/GeoExt/data/store/Features.test.js

```diff
@@ -36,47 +39,73 @@ describe('GeoExt.data.store.Features', function() {
             store = Ext.create('GeoExt.data.store.Features', {features: coll});
         });
         afterEach(function() {
+            if (store.destroy) {
+                store.destroy();
+            }
             store = null;
             coll = null;
         });
 
-        it('constucts an instance of GeoExt.data.store.Features', function() {
+        it('constructs an instance of GeoExt.data.store.Features', function() {
             expect(store).to.be.an(GeoExt.data.store.Features);
         });
-        it('constucts the store with the right amount of records', function() {
+        it('constructs the store with the right amount of records', function() {
             expect(store.count()).to.be(1);
         });
-        it('constucts the store with the correct FC reference', function() {
+        it('constructs the store with the correct FC reference', function() {
             expect(store.getFeatures()).to.be(coll);
         });
 
     });
 
     describe('constructor (with layer)', function() {
-        var layer,
+        var div,
+            map,
+            layer,
             store;
         beforeEach(function() {
+            div = document.createElement('div');
+            document.body.appendChild(div);
+            map = new ol.Map({
+                target: div
+            });
```

Comment:
If there is no map that has the layer, it impossible to test whether the number of layers of the map changes when the store (configured with that layer) is destroyed. See `it('doesn\'t remove a passed layer once the store is destroyed', …`